src/profile/external-identities/ExternalIdentities.tsx

```diff
@@ -12,23 +12,33 @@ type ExternalIdentitiesProps = {
 
 export const ExternalIdentities = (props: ExternalIdentitiesProps): ReactNode => {
   const { queryParams } = props;
+  const providers = getConfig().externalCreds?.providers;
+  const desiredOrder: OAuth2ProviderKey[] = [
+    'ras',
+    'fence',
+    'dcf-fence',
+    'kids-first',
+    'anvil',
+    'github',
+    'era-commons',
+  ];
+  const sortedProviders = providers.sort((a: any, b: any) => desiredOrder.indexOf(a) - desiredOrder.indexOf(b));
 
   return (
     <PageBox role='main' style={{ flexGrow: 1 }} variant={PageBoxVariants.light}>
-      <NihAccount nihToken={queryParams?.['nih-username-token']} />
-      {getConfig()
-        .externalCreds?.providers.filter((p) => p !== 'github')
+      {sortedProviders
+        .filter((p: any) => p !== 'github')
```

Comment:
I realize you are maintaining the existing logic here. That said, could you simplify the github handling here by using a filter of
```ts
        .filter((p: any) => p !== 'github' || userHasAccessToEnterpriseFeature('github-account-linking'))
```
and removing the special-casing down at line 38?

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5259/commits/5a458e3ac636de0bdfaaaf12d201e5517489f2ae